client/src/app/components/employee-settings/employee-settings.component.html

```diff
@@ -13,12 +13,12 @@
 
           <mat-form-field>
             <mat-label for="First Name"></mat-label>
-            <input matInput id="first_name" type="text" [disabled]="true" [value]="employee.first_name"/>
+            <input matInput id="first_name" type="text" [disabled]="!isLoggedInUser"  [value]="employee.first_name"/>
           </mat-form-field>
 
           <mat-form-field>
             <mat-label for="Last Name"></mat-label>
-            <input matInput id="last_name" type="text" [disabled]="true" [value]="employee.last_name"/>
+            <input matInput id="last_name" type="text" [disabled]="!isLoggedInUser" [value]="employee.last_name"/>
```

Comment:
What's the reasoning behind this change?

Reply:
The user must be able to change his first name or last name if he wishes.

Reply:
Didn't we agree to something else in the previous PR?

Reply:
We agreed for it regarding a SA editing other users and the functionality is the same, but we didn't mention the case of editing its own profile and this is where it is changing.

case1:
<img width="1387" alt="Screen Shot 2020-11-01 at 8 47 32 PM" src="https://user-images.githubusercontent.com/35696935/97822316-78540d80-1c83-11eb-8257-f6564ac5da0d.png">

case2:
<img width="1423" alt="Screen Shot 2020-11-01 at 8 47 47 PM" src="https://user-images.githubusercontent.com/35696935/97822330-8144df00-1c83-11eb-8db3-a449a65fde7d.png">

Reply:
yeah, I know, I just don't get why it should be different for the user's own profile. BTW, it should still be disabled when the edit mode is off (like in your picture).

TBH, let's just keep this simple. For now, let's just allow everyone to edit the first name and password (which is contradictory to my previous point in the other PR). The API allows it anyway, so it's not like it's secure or anything. 

If it needs to be changed in the future, then we'll just modify it then. For now let's just add it to the question bank that we have for Ajay (along with how we should handle passwords).